helpers/objects.js

```diff
@@ -0,0 +1,28 @@
+function singleSort(arrayOfObjs, sortProperty1) {
+	return null;
+}
+
+function doubleSort(arrayOfObjs, sortProperty1, sortProperty2) {
+	return arrayOfObjs.sort(function(object1, object2) {
```

Comment:
There's no return for when properties are equal

Reply:
Fixed